nixos/tests/redmine.nix

```diff
@@ -0,0 +1,40 @@
+import ./make-test.nix ({ pkgs, lib, ... }:
+{
+  name = "redmine";
+  meta.maintainers = [ lib.maintainers.aanderse ];
+
+  machine =
+    { config, pkgs, ... }:
+    { services.mysql.enable = true;
+      services.mysql.package = pkgs.mariadb;
+      services.mysql.ensureDatabases = [ "redmine" ];
+      services.mysql.ensureUsers = [
+        { name = "redmine";
+          ensurePermissions = { "redmine.*" = "ALL PRIVILEGES"; };
+        }
+      ];
+
+      services.redmine.enable = true;
+      services.redmine.database.socket = "/run/mysqld/mysqld.sock";
+      services.redmine.plugins = {
+        redmine_env_auth = builtins.fetchurl {
```

Comment:
With `pkgs.fetchurl` this test would be runnable on ofborg, I hope…

Reply:
Out of curiosity... any reason why that is? I haven't found a great source of documentation for the Nix standard library with explanations on when I should use function A vs function B (I've noticed several functions in the builtin namespace have other versions...).

I've made the changes so hopefully ofborg will build now.

Reply:
The builtin version works during evaluation time, the package version works as one of the build steps.